discover/portal/articles/XX-segmentation-and-personalization/02-creating-segments.markdown

```diff
@@ -0,0 +1,54 @@
+# Creating User Segments
```

Comment:
@epgarcia, fyi

Reply:
I'm apparently very bad at git branch management. :) I keep getting things crossed up this week.